kubernetes/loculus/silo_import_job.sh

```diff
@@ -148,6 +151,48 @@ download_data() {
   echo
 }
 
+# Generate the preprocessing config file with the lineage file for the current pipeline version.
+# the lineage definition file needs to be downloaded first.
+prepare_preprocessing_config() {
+  rm -f $lineage_definition_file $preprocessing_config_file_merged
+
+  if [[ -z "$LINEAGE_DEFINITIONS" ]]; then
+    echo "No LINEAGE_DEFINITIONS given, nothing to configure;"
```

Comment:
I guess this probably means that if someone configures the metadata type to lineage but doesn't define a lineageSystem, that will crash SILO. But to me that feels acceptable

Reply:
Ah no, maybe that's not even possible